### ISSUE TITLE
Readme link to new project is broken 

### ISSUE THREAD

**Issue by citadelgrad** [invertase/react-native-firebase-starter#146]
This link is broken in the Readme. If I could find the correct link, I'd have provided a pull request. 

For v6 onwards please follow the [new projects guide](https://invertase.io/oss/react-native-firebase/quick-start/new-project).

**Comment by mikehardy** [comment]
Yeah there's no real starter project anymore but I do maintain a demonstrator that I (mostly) use to prove things work from a clean state for people with issues https://github.com/mikehardy/rnfbdemo/blob/master/make-demo-v6.sh
